acceptance/bundle/summary/resolve-config/targets/default.yml

```diff
@@ -0,0 +1,4 @@
+targets:
+  default:
+    mode: development
+    default: true
```

Comment:
This is normally an error which is returned when resolving the target:

https://github.com/databricks/cli/blob/afc308e92cdf05291e1d1de5f048f18713eec176/bundle/config/mutator/select_default_target.go#L45

Are we intentionally supporting this case?